include/julea-internal.h

```diff
@@ -35,6 +35,12 @@ G_BEGIN_DECLS
 
 #define J_CRITICAL(format, ...) g_critical("%s:%s: " format, G_STRLOC, G_STRFUNC, __VA_ARGS__);
 
+#define J_WARNING(format, ...) g_warning("%s:%s: " format, G_STRLOC, G_STRFUNC, __VA_ARGS__);
+
+#define J_INFO(format, ...) g_info("%s:%s: " format, G_STRLOC, G_STRFUNC, __VA_ARGS__);
+
+#define J_DEBUG(format, ...) g_warning("%s:%s: " format, G_STRLOC, G_STRFUNC, __VA_ARGS__);
+
 /* FIXME j_sync() for benchmarks */
```

Comment:
I was actually planning to get rid of the custom logging functions. :smiley:
Maybe we can provide our own log handler?

Reply:
I would prefer an own log handler since the g_log one is in my taste way to verbose
`** (julea-server:11902): WARNING **: 15:06:39.106: ../server/server.c:121:jd_on_run: Message 14`

something like below would be better in my opinion
`julea-server:WARNING: ../server/server.c:121:jd_on_run: Message 14`

Either way we should generalize the logging interface of julia. As of now its a mishmash of J_CRITICAL, g_log, g_prints, fprints or even no logging at all in some fatal error scenarios. This sometimes makes it pretty difficult to understand why something has failed in tests and/or running julea.